clusterBuildStrategy/buildpacks/buildpacks.yaml

```
@@ -0,0 +1,234 @@
+---
+apiVersion: shipwright.io/v1alpha1
+kind: ClusterBuildStrategy
+metadata:
+  name: buildpacks
+spec:
+  volumes:
+    - name: layers-dir
+      emptyDir: {}
+    - name: empty-dir
+      emptyDir: {}
+    - name: cache-dir
+      emptyDir: {}
+  parameters:
+    - name: cnb-platform-api
+      description: Platform API Version supported
+      default: "0.12"
+    - name: cnb-builder-image
+      description: Builder image containing the buildpacks
+      default: ""
+    - name: cnb-lifecycle-image
+      description: The image to use when executing Lifecycle phases.
+      default: "docker.io/buildpacksio/lifecycle:0.17.0"
+    - name: app-image
+      description: The name of where to store the app image.
+    - name: run-image
+      description: Reference to a run image to use.
+      default: ""
+    - name: cache-image
+      description: The name of the persistent app cache image (if no cache workspace is provided).
+      default: ""
+    - name: cache-dir-name
+      description: Directory to cache files
+      default: cache
+    - name: process-type
+      description: The default process type to set on the image.
+      default: ""
+    - name: source-subpath
+      description: A subpath within the `source` input where the source to build is located.
+      default: ""
+    - name: env-vars
+      type: array
+      description: Environment variables to set during _build-time_.
+      defaults: []
+    - name: platform-dir
+      description: The name of the platform directory.
+      default: empty-dir
+    - # Update the user id to use new paketo builder image till we can use: https://github.com/shipwright-io/community/blob/main/ships/0036-runAs-for-supporting-steps.md
+      name: user-id
+      description: The user ID of the builder image user.
+      default: "1001"
+    - name: group-id
+      description: The group ID of the builder image user.
+      default: "1000"
+    - name: user-home
+      description: Absolute path to the user's home directory.
+      default: /tekton/home
+    - name: cache-pvc-name  # Custom parameter for cache PVC name
+      description: Name of the Persistent Volume Claim for cache
+      default: "ws-pvc"
+  steps:
```

```suggestion
  buildSteps:
```
The `spec.steps` in v1beta1 is actually `spec.buildSteps` in v1alpha1.
I tried using buildSteps, and then the build fails with ConfigCreateError. (Detailed message: container has runAsNonRoot and image will run as root).

When I applied the CBS as is here, the taskRun has just one step (source-default) & nothing else being performed by the containers.

@ayushsatyam146 can you please confirm if you are also observing the same behavior?